.travis.yml

```diff
@@ -88,7 +87,6 @@ script:
     ulimit -c unlimited || :
   - |
     archery docker run \
-      -e ARROW_CSV=${ARROW_CSV:-ON} \
```

Comment:
Why remove this?

Reply:
I was suggested to remove a line at `archery docker run` in [the previous PR](https://github.com/apache/arrow/pull/7142#discussion_r422574426).

Reply:
I am neutral on this.

Reply:
Hmm, fair enough.

Reply:
We don't need it if we don't need to change the default `-DARROW_CSV`.
If we have multiple default definitions, they will confuse us.